src/MDMAnimationTraits.h

```diff
@@ -100,6 +101,13 @@ MDM_SUBCLASSING_RESTRICTED
                            repetition:(nullable id<MDMRepetitionTraits>)repetition
     NS_DESIGNATED_INITIALIZER;
 
+/**
+ Initializes the instance with the provided legacy C-style animation trait structure.
+
+ @param timing A legacy C-style representation of animation traits.
+ */
+- (nonnull instancetype)initWithMotionTiming:(MDMMotionTiming)timing;
```

Comment:
Should this be in a category for easy removal later? Or at least to keep track of the "transitional" code?

Reply:
Good call. Moved.